src/bidiMapper/domains/context/BrowsingContextImpl.ts

```diff
@@ -664,7 +664,10 @@ export class BrowsingContextImpl {
         break;
     }
 
-    return {};
+    return {
+      navigation: this.loaderId ?? null,
```

Comment:
As per https://chromedevtools.github.io/devtools-protocol/tot/Page/#method-navigate:

> Loader identifier. This is omitted in case of same-document navigation, as the previously committed loaderId would not change.

My understanding is that `reload` does not alter this ID. Same below for the URL.